includes/options/class-amp-reader-theme-rest-controller.php

```diff
@@ -27,43 +27,10 @@ final class AMP_Reader_Theme_REST_Controller extends WP_REST_Controller {
 	 */
 	public function __construct( $reader_themes ) {
```

Comment:
We'll want to add some HTTP mocks for the WordPress.org response in `\Test_Reader_Theme_REST_Controller::test_get_items()`, except if `in_array( 'external-http', $_SERVER['argv'], true )`. This can be seen in other tests, like `AMP_Crowdsignal_Embed_Handler_Test`.